fastjsonschema/generator.py

```diff
@@ -97,12 +102,14 @@ def global_state_code(self):
 
         if not self._compile_regexps:
             return '\n'.join(self._extra_imports_lines + [
+                'from decimal import Decimal',
```

Comment:
I think its included twice if added here and in _extra_imports_lines.

Reply:
Removed